### ISSUE TITLE
How to train Joint regressor for a different dataset

### ISSUE THREAD

**Issue by zhipenggong** [hongsukchoi/Pose2Mesh_RELEASE#72]
Hi @hongsukchoi 

I would like to train the model with some other datasets, could you please tell me how to train Joint regressor for a different dataset, 

Thank you very much and looking forward to your reply

**Comment by hongsukchoi** [comment]
Hi, do you mean you want to train a PoseNet? What is the joint format of that dataset?

**Comment by zhipenggong** [comment]
Yes, it's a mocap dataset captured by us, but with different joints positions from human3.6m and coco. We would like to train a Joint regressor to convert SMPL model to the joints for the comparison.  Could you please help give us some guidance? Thanks a lot.

**Comment by hongsukchoi** [comment]
Hi, 

Sorry for the late reply. I remember we manually picking the location from the template posed mesh.
